lib/active_encode/technical_metadata.rb

```diff
@@ -22,5 +22,12 @@ module TechnicalMetadata
       attr_accessor :audio_bitrate
       attr_accessor :video_bitrate
     end
+
+    # Assign values from a Hash
+    def assign_tech_metadata tech_md
```

Comment:
I'm not really a fan of this and think it would be better to have `ActiveEncode::Base` include `ActiveModel::AttributeAssignment` instead.
https://api.rubyonrails.org/classes/ActiveModel/AttributeAssignment.html